apps/frontend/src/app/5_pages/ZeroPage/ZeroPage.tsx

```diff
@@ -62,63 +65,204 @@ export const ZeroPage: FC = () => {
+
   return (
     <div className="container max-w-7xl mt-24">
-      {account ? (
-        <>
-          <Heading>Example</Heading>
-          <div className="flex flex-row justify-start items-center text-black gap-8 mt-8">
-            <div className="bg-gray-30 p-3">
-              <div>Debt</div>
-              <div>{trove?.debt.toString() ?? '0'}</div>
-            </div>
-            <div className="bg-gray-30 p-3">
-              <div>Collateral</div>
-              <div>{trove?.collateral.toString() ?? '0'}</div>
-            </div>
-          </div>
-
-          <Button text="Adjust" onClick={toggle} className="mt-8" />
-          <Button
-            text="Close Line of Credit"
-            onClick={toggleClosePopup}
-            className="mt-8 ml-4"
-          />
-        </>
-      ) : (
-        <>
-          <Button text="Connect first...." onClick={connectWallet} />
-          <hr />
-          <br />
-          <SystemStats />
-        </>
-      )}
-      <LOCChart />
-
-      <Dialog width={DialogSize.sm} isOpen={open} disableFocusTrap>
-        <DialogHeader title="Adjust" onClose={toggle} />
-        <DialogBody>
-          <AdjustCreditLine
-            collateralValue={trove?.collateral.toString() ?? '0'}
-            creditValue={trove?.debt.toString() ?? '0'}
-            onSubmit={noop}
-            rbtcPrice={btcPrice}
-            fees={fees}
-          />
-        </DialogBody>
-      </Dialog>
-
-      <Dialog width={DialogSize.sm} isOpen={openClosePopup} disableFocusTrap>
-        <DialogHeader title="Close" onClose={toggleClosePopup} />
-        <DialogBody>
-          <CloseCreditLine
-            onSubmit={noop}
-            creditValue={trove?.debt.toString() ?? '0'}
-            collateralValue={trove?.collateral.toString() ?? '0'}
-            availableBalance={zusdBalance}
-          />
-        </DialogBody>
-      </Dialog>
+      <React.Suspense fallback={<p>Loading stuff...</p>}>
+        <Await resolve={deferedData} errorElement={<p>Error loading stuff!</p>}>
+          {([price, fees]: [string, Fees]) => (
+            <>
+              {account ? (
+                <>
+                  {trove?.debt?.gt(0) ? (
+                    <LOCStatus
+                      collateral={Number(trove?.collateral ?? 0)}
+                      debt={Number(trove?.debt ?? 0)}
+                      cRatio={
+                        ((Number(trove?.collateral ?? 0) * Number(price)) /
+                          Number(trove?.debt ?? 0)) *
+                        100
+                      }
+                      debtSymbol={'ZUSD'}
+                      onAdjust={toggle}
+                      onClose={toggleClosePopup}
+                    />
+                  ) : (
+                    <>
+                      <Paragraph>****** It should be banner....</Paragraph>
+                      <div className="mt-8">
+                        <Button
+                          text={t('zeroPage.loc.open')}
```

Comment:
I am not sure if we want to use translations like this, you don't have autocomplete in this case. We should discuss it with the team.

cc @soulBit